lambda_functions/v1/functions/lpa_codes/app/api/code_generator.py

```diff
@@ -73,7 +73,9 @@ def get_codes(database, key=None, code=None):
 
     table = database.Table(lpa_codes_table())
 
-    return_fields = "lpa, actor, code, active, last_updated_date, dob, expiry_date"
+    return_fields = (
+        "lpa, actor, code, active, last_updated_date, dob, expiry_date, generated_date"
+    )
```

Comment:
I had to add brackets and new line to reduce the line length to avoid the linting error